src/book.py

```diff
@@ -16,6 +16,7 @@
 
 import csv
 import datetime
+from dateutil.tz import tzutc
```

Comment:
I had to explicitly set the timezone to UTC on my timestamps. I guess you should also check if your timestamps match what you expect.